python/plugins/processing/tools/general.py

```diff
@@ -53,11 +54,13 @@ def algorithmHelp(id):
         print('Input parameters')
         print('----------------')
         for p in alg.parameterDefinitions():
+            if p.flags() & QgsProcessingParameterDefinition.FlagHidden:
+                continue
             print(f'\n{p.name()}: {p.description()}')
             if p.help():
                 print(f'\n\t{p.help()}')
 
-            print(f'\n\tParameter type:\t{p.__class__.__name__}')
+            print(f'\n\tParameter type: {p.__class__.__name__}')
```

Comment:
why this change?

Reply:
It looks like such tabulation character is badly displayed in the Python console on Windows
![image](https://github.com/qgis/QGIS/assets/16253859/6ac234df-b620-40da-8861-39d61fc13f04)
and I think it's useless and in other strings the colon is always followed by a space.
If OK, I would also remove the double space in 
https://github.com/qgis/QGIS/blob/11f083b6c0eb67c8d119c9c8a631ccac5d2489e5/python/plugins/processing/tools/general.py#L81-L82
Otherwise I can restore the tabulation.

Reply:
Hm, odd -- because the lines just above that use tab characters!

My gut feeling is that this is just because we (incorrectly) try to parse any outputs in the Python console as python code itself. I'd say we leave the current behavior and fix the issue in the console instead...

Reply:
@nyalldawson, OK. Tab restored and extra space not removed.